src/main/java/io/appium/java_client/android/GsmCallActions.java

```diff
@@ -0,0 +1,18 @@
+package io.appium.java_client.android;
+
+public enum GsmCallActions {
+    CALL("call"),
+    ACCEPT("accept"),
+    CANCEL("cancel"),
+    HOLD("hold");
+
+    private final String gsmcall;
+
+    GsmCallActions(String gsmcall) {
```

Comment:
this constructor is redundant. All enum elements have `name()` and `ordinal()` properties. That's pretty all you need. Same comment to other enums